config/services.php

```diff
@@ -17,6 +17,7 @@
             ->args([
                 param('kernel.project_dir'),
                 abstract_arg('path to source Tailwind CSS file'),
+                abstract_arg('path to source Tailwind CSS config file'),
```

Comment:
Is it really called as "source" Tailwind CSS config file?

I think correctly will be just:
```suggestion
                abstract_arg('path to Tailwind CSS config file'),
```

Right?